web/src/app/admin/SlackActions.tsx

```diff
@@ -63,7 +63,7 @@ export default function SlackActions(): JSX.Element {
           {
             label: 'App Manifest',
             handleOnClick: () => {
-              getManifest()
+              commit()
```

Comment:
This causes a re-fetch (since it's `useQuery`); the original behavior of this is code was not to send the query until clicked.

However, there's no reason to wait; this handler can just set `setShowManifest`
```suggestion
```

Reply:
Gotcha, hard to tell when there's no comments for business logic